### DISCUSSION TITLE
Stop events when a form is displayed

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1905] [Q&A]
Hello,

I have events to draw an element and when is draw a form is display. My problem is that I can continue to draw an element when the form is display. Is there a solution  to stop all events when a form is open. I try paper.off but doesn't work...

Thank you

**Comment by kumilingus** [comment]
```js
paper.undelegateEvents() // stop triggering the paper events
paper.delegateEvents() // start triggering the paper events
```
